wpeverest/user-registration
# UR-96 Feature - Form templates

### All Submissions:






### Changes proposed in this Pull Request:
Previously we didn't have a way for users to select an already listed form template so that a form can be created as a replica of the selected form template while adding or creating a new form. This PR provides this feature.

### How to test the changes in this Pull Request:

1. Click on User Registration -> Add New or Create New Form after the Getting Started page is finished.
2. Select one of the templates.
3. Check if the form is created according to the template or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Form templates.
